app/services/event_log/move_runner.rb

```diff
@@ -15,6 +15,8 @@ def call
           move.to_location = event.to_location
         when 'complete'
           move.status = Move::MOVE_STATUS_COMPLETED
+        when 'lockout'
```

Comment:
if there is no action for `lockout` maybe we can leave it out of here and not `run_event_logs` in the controller? (as this will appear in the event log itself?)

Reply:
hmm... I like the explicitness here where it is clear that no action is required for lockouts. If it were absent it might appear to be a mistake.

As for removing the call to the event runner from the controller, I think its better for the event runner to decide how to process (or not) the events, rather than the controller.

Reply:
Sorry meant removing `run_event_logs` from the action as in: 
```
def lockouts
   create_event('lockout')
   render status: :no_content
end
```
But if your keeping it to be explicit that's fine

Reply:
yep, I see what you mean - but I think it should be for the event runner to decide whether it has work to do (or not), not for the controller.